app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -8,7 +9,7 @@ related_content:
     Who do you want to teach? : "/is-teaching-right-for-me/who-do-you-want-to-teach"
 promo_content:
     - content/non-uk-teachers/promos/adviser-promo-ukraine
-navigation: 20.50
+navigation: 20.40
 navigation_title: Ukrainian teachers and trainees coming to the UK
 navigation_description: Find information and advice for Ukrainian citizens seeking teaching jobs and training places in England.
 date: "2022-06-24"
```

Comment:
@laurahermionetennant92 As the site is focused on getting people into teacher training can we switch these round so the title and description have trainees first and then teachers.